rustler_mix/lib/rustler.ex

```diff
@@ -170,5 +170,14 @@ defmodule Rustler do
   end
 
   @doc false
-  def rustler_version, do: "0.33.1"
+  def rustler_version do
+    # Retrieve newest version or fall back to hard-coded one
+    Req.get!("https://crates.io/api/v1/crates/rustlaer").body
```

Comment:
Typo

Reply:
Oh yeah, I used that to test the error case :)
Thanks! :)